Sources/UIKit/UIKit+AdaptableTextContainerSupport.swift

```diff
@@ -37,8 +32,8 @@ extension UIApplication {
     @objc internal func bon_notifyContainedAdaptiveContentSizeContainers(fromNotification notification: NSNotification) {
         // First notify the app delegate if it conforms to AdaptableTextContainer.
         if let container = delegate, let traitCollection = container.window??.traitCollection {
-            if container.responds(to: #selector(AdaptableTextContainer.adaptText(forTraitCollection:))) {
-                container.perform(#selector(AdaptableTextContainer.adaptText(forTraitCollection:)), with: traitCollection)
+            if NSObject.responds(to: #selector(AdaptableTextContainer.adaptText(forTraitCollection:))) {
+                NSObject.perform(#selector(AdaptableTextContainer.adaptText(forTraitCollection:)), with: traitCollection)
```

Comment:
Curious why the change to `NSObject` here? Also, would probably reduce the if to `if let traitCollection = delegate?.window??.traitCollection {` since you're no longer using container

Reply:
I don't know why this was changed. Guessing the migrator made a mistake. I can't think of any reason to keep it, and it looks like it would break things, so I'm going to change it back.